pact/genesis/testnet/allocations.yaml

```diff
@@ -4,7 +4,7 @@ code: |-
   (coin.create-allocation-account "allocation00" (time "1900-10-15T18:00:00Z") "allocation00" 1000000.0)
 
   (define-keyset "allocation01" (read-keyset "allocation01"))
-  (coin.create-allocation-account "allocation01" (time "2019-10-31T18:00:00Z") "allocation01" 1000000.0)
+  (coin.create-allocation-account "allocation01" (time "2020-10-31T18:00:00Z") "allocation01" 1000000.0)
```

Comment:
Does this mean our tests will spontaneously start failing in a year?

Reply:
By the sounds of it, yes.